lib/l10n_utils/__init__.py

```diff
@@ -51,10 +49,15 @@ def render_to_string(template_name, context=None, request=None, using=None, ftl_
     return loader.render_to_string(template_name, context, request, using)
 
 
-def redirect_to_best_locale(request: HttpRequest, translations: List[str]) -> HttpResponseRedirect:
+def redirect_to_best_locale(request, translations):
     # Note that translations is list of locale strings (eg ["en-GB", "ru", "fr"])
     lang = get_best_translation(translations, get_accept_languages(request))
-    response = HttpResponseRedirect("/" + "/".join([lang, split_path(request.get_full_path())[1]]))
+    return redirect_to_locale(request, lang)
+
+
+def redirect_to_locale(request, locale, permanent=False):
+    redirect = HttpResponsePermanentRedirect if permanent else HttpResponseRedirect
```

Comment:
Naming micro-nit: maybe rename to `redirect_class` or `RedirectClass` or something, just because `redirect` is also a helper func from `django.shortcuts` so is commonly seen in code.